packages/js-dpp/lib/stateTransition/errors/PublicKeySecurityLevelNotMetError.js

```diff
@@ -0,0 +1,35 @@
+const DPPError = require('../../errors/DPPError');
+
+class PublicKeySecurityLevelNotMetError extends DPPError {
+  /**
+   *
+   * @param {IdentityPublicKey} publicKey
```

Comment:
You can pass only raw data that can be encoded with cbor, transferred to the client and decoded there. So please don't put unnecessary information to arguments. If you need something in addition, please you setters.